net.sourceforge.xournal.json

```diff
@@ -0,0 +1,114 @@
+{
+  "app-id": "net.sourceforge.xournal",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "xournal",
+  "rename-icon": "xournal",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=host"
+  ],
+  "modules": [
+    {
+      "name": "libart",
+      "rm-configure": true,
+      "cleanup": [
+        "/bin",
+        "/share"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libart_lgpl/2.3/libart_lgpl-2.3.21.tar.gz",
+          "sha256": "2a10b1c884bb37c7bd5c5eba59dbd0d28bf68e64a42d7f2083c836f60f8e98fa"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "rm -f config.guess"
+          ]
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+    {
+      "name": "libgnomecanvas",
+      "rm-configure": true,
+      "cleanup": [
+        "/share"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libgnomecanvas/2.30/libgnomecanvas-2.30.3.tar.gz",
+          "sha256": "a8ca85e734ab03ecf1fba7d99e01ae2541d0df539c69a7da9414cde928c116da"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "rm -f config.guess"
+          ]
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+    {
+      "name": "poppler",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_INSTALL_INCLUDEDIR=/app/include",
+        "-DENABLE_LIBOPENJPEG=none"
+      ],
+      "cleanup": [
+        "/bin",
+        "/share"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://poppler.freedesktop.org/poppler-0.64.0.tar.xz",
+          "sha256": "b21df92ca99f78067785cf2dc8e06deb04726b62389c0ee1f5d8b103c77f64b1"
+        }
+      ]
+    },
+    {
+      "name": "xournal",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/xournal/xournal-0.4.8.2016.tar.gz",
+          "sha256": "b25898dbd7a149507f37a16769202d69fbebd4a000d766923bbd32c5c7462826"
+        },
+        {
+          "type": "file",
+          "path": "appdata.xml"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "mkdir -p /app/share/applications/",
+            "cp xournal.desktop /app/share/applications/net.sourceforge.xournal.desktop",
+            "mkdir -p /app/share/icons/hicolor/512x512/apps",
+            "cp pixmaps/xournal.png /app/share/icons/hicolor/512x512/apps/xournal.png",
```

Comment:
No need for a rename icon. You can just copy the icon with the correct icon name here. And I think it would be better to use install instead of mkdir & cp. This way you can set the right permissions of the files too.

Reply:
I found out, that with renaming the icon, the desktop file get's fixed using the right icon.

Reply:
Fixed that

Reply:
Thanks! :+1:

Reply:
What's left for approval?